style.css

```diff
@@ -1,3 +1,5 @@
+/* stylelint-disable CssSyntaxError */
+
 /* For mobile phones */
 @media only screen and (max-width: 600px) {
```

Comment:
- [ ] It is advisable to build your project mobile first and use @media only screen and (min-width: 765px) to build for desktop.